fonts/iconoir.css

```diff
@@ -0,0 +1,3694 @@
+/*!
+ * Iconoir
+ * Copyright (c) 2021 Luca Burgio - https://iconoir.com
+ * License - https://github.com/lucaburgio/iconoir/blob/master/LICENSE (Code: MIT License)
+ * CSS-File created by Till Esser @Wiwaltill
+ */
+
+@font-face {
+    font-family: "Iconoir";
+    src: url("https://raw.githubusercontent.com/lucaburgio/iconoir/master/fonts/iconoir.eot");
+    src: url("https://raw.githubusercontent.com/lucaburgio/iconoir/master/fonts/iconoir.eot?#iefix") format("embedded-opentype"),
+         url("https://raw.githubusercontent.com/lucaburgio/iconoir/master/fonts/iconoir.svg") format("svg"),
+         url("https://raw.githubusercontent.com/lucaburgio/iconoir/master/fonts/iconoir.ttf") format("truetype"),
+         url("https://raw.githubusercontent.com/lucaburgio/iconoir/master/fonts/iconoir.woff") format("woff");
+    font-weight: normal;
+    font-style: normal;
+}
+
+
+span[class^="iconoir-"]:before, span[class*=" iconoir-"]:before{
+    font-family: "Iconoir" !important;
+}
+
+.iconoir{
+    -moz-osx-font-smoothing: grayscale;
+    -webkit-font-smoothing: antialiased;
+    display: inline-block;
+    font-style: normal;
+    font-variant: normal;
+    text-rendering: auto;
+    line-height:1
+}
```

Comment:
I think instead of using a separate class, this part will be better to be placed in the `:before` scope.

Reply:
Thanks for the tip! This class should actually also be defined differently.